backend/src/main/kotlin/app/ehrenamtskarte/backend/application/webservice/EakApplicationMutationService.kt

```diff
@@ -29,4 +29,11 @@ class EakApplicationMutationService {
         return true
     }
 
+    @GraphQLDescription("Deletes the application with specified id")
+    fun deleteApplication(
+        context:GraphQLContext, applicationId: Int
+    ): Boolean {
+        context.enforceSignedIn()
```

Comment:
This throws an exception if not signed in right?

Reply:
Yep, I guess so